### PR TITLE
Replace reqwest by fetch API

### LINKED ISSUE #66: Use browser's fetch API so reqwest can be dropped (required by AMO)

### PR DESCRIPTION
**Pull request by LEDfan**
Fixes #66

### CODE REVIEW

File: background/init.js [hunk 1]
```diff
@@ -44,14 +44,16 @@ function init() {
                     }
                 });
             }
-        }).catch(function(ss) {
+        }).catch(function(err) {
+            console.log(err);
             console.log("Failed to initialize Secure Storage, not associating with keepass!");
-            Keepass.setSecureStorage(ss);
+            browser.notifications.create({
```
Comment (RobinJadoul) [inline background/init.js]: How does this work with the new localization? Is it already integrated there?

File: background/keepass.js [hunk 2]
```diff
@@ -240,67 +224,56 @@ Keepass.getLogins = function (url, callback) {
...
-
-                        }).catch(function(resp) {
-                            console.log("RetrieveCredentials for " + url + " rejected");
-
+                        });
+                    }
+                    promiseChain.then(function() {
+                        // decrypted all entries
+                        if (decryptedEntries.length === 0) {
                             browser.notifications.create({
```
Comment (RobinJadoul) [inline background/keepass.js]: Localization?

File: background/keepass.js [hunk 3]
```diff
@@ -240,67 +224,56 @@ Keepass.getLogins = function (url, callback) {
...
                             });
-                        });
-                    }
+
+                        }
+                    });
+                }).catch(function(resp) {
+                    console.log("RetrieveCredentials for " + url + " rejected");
+
+                    browser.notifications.create({
```
Comment (RobinJadoul) [inline background/keepass.js]: l10n

File: background/keepass.js [hunk 4]
```diff
@@ -240,67 +224,56 @@ Keepass.getLogins = function (url, callback) {
...
+                        type: "basic",
+                        message: "Problem getting logins from your Keepass database, have you associated with this database?",
+                        iconUrl: browser.extension.getURL("icons/keywi-96.png"),
+                        title: "Keywi"
+                    });
+                });
+            }).catch(function(err) {
+                browser.notifications.create({
```
Comment (RobinJadoul) [inline background/keepass.js]: l10n

File: README.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ Minimalistic Keepass plugin using Web Extensions and KeepassHTTP.
  - No editing of the Keepass database
  - Using the Firefox web extensions API, to support Electrolysis
  - Support Keepass, later maybe [Pass](https://www.passwordstore.org/)
- - Minimal amount of libraries (aes, cryptoheplers and reqwest)
+ - Minimal amount of libraries (aes and cryptoheplers)
```
Comment (RobinJadoul) [inline README.md]: typo: cryptoheplers

**Comment by LEDfan** [comment]
l10n will be handled when #62 is merged (this can then be rebased)

**Comment by LEDfan** [comment]
@RobinJadoul rebased and add a new translation string, please have a final look 😄 (I'll squash when merging)

**Pull request merged**